onebusaway-android/src/main/java/org/onebusaway/android/ui/RouteInfoListFragment.java

```diff
@@ -467,15 +464,7 @@ private void setHeader(ObaRouteResponse routeInfo, boolean addToDb) {
             agencyText.setText(mRouteInfo.getAgency().getName());
 
             if (addToDb) {
-                ContentValues values = new ContentValues();
-                values.put(ObaContract.Routes.SHORTNAME, shortName);
-                values.put(ObaContract.Routes.LONGNAME, longName);
-                values.put(ObaContract.Routes.URL, url);
```

Comment:
My new method does not have `URL` in the ContentValues. Is it needed?

Reply:
Yes, let's keep route `URL`.  Even if it's not used today it will likely be used in a future favorite routes list.